### PR TITLE
Improve default tours

### PR DESCRIPTION
**Pull request by bgruening**
This PR will improve the two default tours we currently ship. I included also a little bit of developer information as comments, in the hope to make it easier to develop new tours.

### CODE REVIEW

File: config/plugins/tours/core.history.yaml [hunk 1]
```diff
@@ -122,11 +127,11 @@ steps:
...
       preclick:
-        -  "#current-history-panel > div.controls > div.search > div > input"
+        - "#current-history-panel > div.controls > div.search > div > input"
 
     - element: "#history-options-button"
       title: "History Options"
-      intro: "In the History menue you will find a lot more usful History options."
+      intro: "In the History menue you will find a lot more useful History options."
       position: "left"
       postclick:
```
Comment (bgruening) [inline config/plugins/tours/core.history.yaml:136]: @guerler @dannon during this step something wired happened. Would be wonderful if you can have a look at it.
We can also remove this last step.

File: config/plugins/tours/core.history.yaml [hunk 2]
```diff
@@ -122,11 +127,11 @@ steps:
       position: "left"
       textinsert: "WWFSMD"
       preclick:
-        -  "#current-history-panel > div.controls > div.search > div > input"
+        - "#current-history-panel > div.controls > div.search > div > input"
 
     - element: "#history-options-button"
       title: "History Options"
-      intro: "In the History menue you will find a lot more usful History options."
+      intro: "In the History menue you will find a lot more useful History options."
```
Comment (nsoranzo) [inline config/plugins/tours/core.history.yaml]: s/menue/menu/

**Comment by bgruening** [comment]
@nsoranzo thanks and fixed!

**Comment by dannon** [comment]
@bgruening Thanks!

**Comment by martenson** [comment]
since this is merged I moved my comment to #1423

**Comment by dannon** [comment]
@martenson Regarding clicking on things -- this is a _hugely_ important point.  Unless the tour is intended to be a totally automatic sit-back-and-watch-it type tour (which we can actually build with this), I really think we should keep automatic clicking and text entry to a minimum, instead using text like "Now click this element to open the upload dialog".  There is a wait state you can apply to a tour step to advance to the next step automatically upon clicking the element in question, which is what I'd prefer using.  People learn more and follow along better when they actually click things and enter text themselves, is my hunch.

That said, this is a major improvement on the previous incarnation, and we can definitely still keep working on it.  The Galaxy101 version I'm doing now does not use our preclick/postclick events, intentionally.

**Pull request merged**